### PR TITLE
14 start page

### PR DESCRIPTION
**Pull request by itggot-alex-henryz**
Created start page

### CODE REVIEW

**Comment by itggot-Filip-Gustavsson** [comment]
Screenshot of how It [looks](https://imgur.com/XJ1ds0n)

File: frontend/src/views/Start.vue [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+        b-container
+            h1 Välkommen till YABS! 
+            br
+            p.info Denna applikation kommer att användas för att underlätta hantering/utlåningen av olika sorters litteratur och utskrivning av Passerkort.
+            br
+            b-row
+                b-col
+                    router-link.link.row-btn(to='/users')
+                        b-button.row-btn(:size="size" :variant="primary") Profil
+            p.info På profil sidan så ser du all relevant information agngående din profil tex antal aktiva lån från dig.
```
Comment (itggot-linus-styren) [inline frontend/src/views/Start.vue]: Fix typo.

File: frontend/src/views/Start.vue [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+            br
+            b-row
+                b-col
+                    router-link.link.row-btn(to='/add')
+                        b-button.row-btn(:size="size" :variant="primary") Lägg till
+            p.info Lägg till delen används för att skapa nya Passerkort, registera nya lån och lägga till nya böcker.
```
Comment (itggot-linus-styren) [inline frontend/src/views/Start.vue]: Swedish isn't German. Use small letter for "Passerkort"

Comment (itggot-Filip-Gustavsson) [reply frontend/src/views/Start.vue]: fixed

**Pull request merged**